filament/src/details/Skybox.h

```diff
@@ -66,6 +68,7 @@ class FSkybox : public Skybox {
     FRenderableManager& mRenderableManager;
     float mIntensity = 0.0f;
     uint8_t mLayerMask = 0x1;
+    SkyboxType mType = Skybox::ENVIRONMENT;
```

Comment:
Why do we store it here?

Reply:
It is unnecessary, fixed in https://github.com/shapr3d/filament/pull/8/commits/91d5e221aec0759129b6da5ad786383348ffd4a2